### PR TITLE
Add Colobot 0.1.12

### PR DESCRIPTION
**Pull request by latere-a-latere**
Part of https://github.com/MarNicGit/winget-pkgs/issues/4

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/771)

### CODE REVIEW

**Comment by MarNicGit** [comment]
Runner says that it timed out during install. Works fine on my machine?

File: manifests/Epsitec/Colobot/0.1.12.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Id: Epsitec.Colobot
+Version: 0.1.12
+Name: "Colobot: Gold Edition"
+Publisher: Epsitec SA, TerranovaTeam
```
Comment (Reset12138) [inline manifests/Epsitec/Colobot/0.1.12.yaml:4]:
```suggestion
Publisher: Epsitec
```
the parent folder should be as same as the Publisher

File: manifests/Epsitec/Colobot/0.1.12.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Id: Epsitec.Colobot
+Version: 0.1.12
+Name: "Colobot: Gold Edition"
```
Comment (Reset12138) [inline manifests/Epsitec/Colobot/0.1.12.yaml:3]:
```suggestion
Name: Colobot: Gold Edition
```
reference
image

**Review by Reset816 — CHANGES_REQUESTED** [review]

File: manifests/Epsitec/Colobot/0.1.12.yaml (hunk 2 above)
Comment (MarNicGit) [reply manifests/Epsitec/Colobot/0.1.12.yaml:3]: This breaks YAML and will not validate. (note the semicolon)

File: manifests/Epsitec/Colobot/0.1.12.yaml (hunk 1 above)
Comment (MarNicGit) [reply manifests/Epsitec/Colobot/0.1.12.yaml:4]: Is this really the issue? The pull request I submitted for Planet Blupi, another title by Epsitec SA, checked out fine: https://github.com/microsoft/winget-pkgs/pull/770

Other manifests, such as [`EpicGames.EpicGamesLauncher`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/EpicGames/EpicGamesLauncher/10.15.2.Yaml), [`GitHub.cli`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/GitHub/cli/0.8.0.yaml) and Microsoft's own [`Microsoft.Edge`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/Microsoft/Edge/81.0.416.77.yaml) also have a publisher that's different from the parent folder.

Looking at the [pipeline](https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=81962&view=logs&jobId=c8020aa5-4aab-5881-691f-21986e02afd0&j=c8020aa5-4aab-5881-691f-21986e02afd0&t=bade86d6-d2be-5ddd-c1b9-7d8db9c4a60d) it appears that the "Installers Scan" task simply ends up timing out.

Comment (Reset12138) [reply manifests/Epsitec/Colobot/0.1.12.yaml:4]: oh you are right. sorry for my wrong suggestion

**Comment by KevinLaMS** [comment]
Sitting Idle.  Kicking it again.  
/AzurePipelines run

**Comment by MarNicGit** [comment]
@KevinLaMS Welp, still not working :/

**Comment by MarNicGit** [comment]
@KevinLaMS

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**